app/src/forms/admin/routes.js

```diff
@@ -18,22 +18,172 @@ routes.use(currentUser);
 //
 // Forms
 //
+
+/**
+ * @openapi
+ * /admin/forms:
+ *  get:
+ *    tags:
+ *      - Admin
+ *    description: This endpoint will fetch all forms under the admin access.
+ *    security:
+ *      - bearerAuth: []
+ *      - basicAuth: []
+ *      - openId: []
+ *    responses:
+ *      '200':
+ *        description: Success
+ *        content:
+ *          application/json:
+ *            schema:
+ *              $ref: '#/components/responses/responseBody/AdminsFormGetEx'
+ *      '403':
+ *        $ref: '#/components/responses/Forbidden'
+ *      '401':
+ *        $ref: '#/components/responses/NoFormAccess'
```

Comment:
Do we need form access for the /admin endpoints? Or do these endpoint bypass the form access for admin users? The old docs don't mention 401 (but they could be wrong!)